src/main/java/frc/robot/RobotContainer.java

```diff
@@ -313,6 +314,7 @@ private void configureButtonBindings() {
 
     // Need to set to use automated movements, should be set in Autonomous init.
     driverController.back().onTrue(new ArmCalibrateCMD(arm, led2023));
+    driverController.a().onTrue(new ArmCalibrateZeroAtHomeCMD(arm, led2023));
```

Comment:
This should be changed to a different button

Reply:
Moved to B.